users/shared/.config/claude/statusline.sh

```
@@ -34,64 +34,64 @@ readonly YELLOW='\033[93m'    # Bright yellow for modified git status
-if [[ "$context_length" -eq 0 ]]; then
-    used_percentage=$(echo "$input" | jq -r '.context_window.used_percentage // empty' 2>/dev/null)
-    context_window_size=$(echo "$input" | jq -r '.context_window.context_window_size // empty' 2>/dev/null)
-
-    if [[ -n "$used_percentage" ]] && [[ "$used_percentage" != "null" ]] && \
-       [[ -n "$context_window_size" ]] && [[ "$context_window_size" != "null" ]]; then
-        # Calculate: context_length = (used_percentage / 100) * context_window_size
-        context_length=$(awk -v pct="$used_percentage" -v size="$context_window_size" \
-            'BEGIN {printf "%.0f", (pct / 100) * size}')
+# Extract data from JSON input using single jq call for performance
+# Use tab as IFS to handle spaces in model names correctly
+IFS=$'\t' read -r model_name current_dir max_context transcript_path <<< "$(echo "$input" | jq -r '[
```

Shellcheck (0.11.0)

[warning] max_context appears unused. Verify use (or export if used externally).

(SC2034)